lib/much-rails.rb

```diff
@@ -1,4 +1,7 @@
 require "much-rails/version"
 
+require "active_support"
+require "active_support/core_ext"
```

Comment:
What is requiring `core_ext` doing for us? You shouldn't have to bring in `core_ext` yes? If so, let's try going forward without this until we have a tangible reason to add it. Cool by you?

Reply:
I could do `active_support/all` instead of `active_support/core_ext`, but I didn't want to bring everything in, so limited it to the core extensions (aka methods) that I think we may want to use in the future. I'm cool to change that if desired.

Reply:
No, don't do `/all`. I want just the bare minimum necessary. Can you drop the `require "active_support"`?

Reply:
I like `active_support/core_ext` for now. I don't like requiring too specific ActiveSupport files as this is more to maintain as Rails upgrades but I think `core_ext` or `all` is pretty safe (I prefer not requiring `all` if we can avoid it.

Reply:
The `require "active_support"` is needed (breaks w/o it), and doesn't load anything by default: https://guides.rubyonrails.org/active_support_core_extensions.html